invite/src/invite_bot.ts

```diff
@@ -115,16 +115,23 @@ export class InviteBot {
     // The user has not been invited by the bot yet, so try and send an invite.
     let invited = false;
     try {
-      invited = await this.github.inviteUser(invite.username);
+      invited = await this.github.inviteUser(invite.username).catch(error => {
```

Comment:
`await`ing a `promise` that rejects will simply throw that rejection as an `Error`, so basically what you wrote here is:
```typescript
try {
  try {
    invited = await this.github.inviteUser(invite.username);
  } catch (error) {
    throw error;
  }
} catch (error) {
  await this.handleError...
```

(TL;DR: don't add the `.catch(...)` part)

Reply:
This was the product of a deep rabbit hole of trying to mock, then catch, then assert errors/rejections in test lol. Fixed.